README.md

```diff
@@ -137,11 +148,146 @@ OTOneToOneCommunicationController *vc = [OTOneToOneCommunicationController oneTo
+
+## Annotation sample code
+
+The `OTAnnotationScrollView` class is the backbone of the annotation features in this Sample.
+
+
+```objc
+self.annotationView = [[OTAnnotationScrollView alloc] init];
+self.annotationView.frame = <# desired frame #>;
+[self.annotationView initializeToolbarView];
+self.annotationView.toolbarView.frame = <# desired frame #>;
+```
+
+If you would like to be annotated on either the entire screen or a specified portion of the screen:
+
+```objc
+self.annotator = [[OTAnnotator alloc] init];
+[self.annotator connectForReceivingAnnotationWithSize:<# desired size #>
+                                    completionHandler:^(OTAnnotationSignal signal, NSError *error) {
+                                        if (signal == OTAnnotationSessionDidConnect){
+                                            self.annotator.annotationScrollView.frame = self.view.bounds;
+                                            [self.view addSubview:self.annotator.annotationScrollView];
+                                        }
+                                    }];
+
+self.annotator.dataReceivingHandler = ^(NSArray *data) {
+    NSLog(@"%@", data);
+};
+```
+
+If you would like to annotate on a remote client's screen:
+
+```objc
+self.annotator = [[OTAnnotator alloc] init];
+[self.annotator connectForSendingAnnotationWithSize:self.sharer.subscriberView.frame.size
+                                completionHandler:^(OTAnnotationSignal signal, NSError *error) {
+    
+                                    if (signal == OTAnnotationSessionDidConnect){
+        
+                                        // configure annotation view
+                                        self.annotator.annotationScrollView.frame = self.view.bounds;
+                                        [self.view addSubview:self.annotator.annotationScrollView];
+
+                                        // self.sharer.subscriberView is the screen shared from a remote client.
+                                        // It does not make sense to `connectForSendingAnnotationWithSize` if you don't receive a screen sharing.
+                                        [self.annotator.annotationScrollView addContentView:self.sharer.subscriberView];
+        
+                                        // configure annotation feature
+                                        self.annotator.annotationScrollView.annotatable = YES;
+                                        self.annotator.annotationScrollView.annotationView.currentAnnotatable = [OTAnnotationPath pathWithStrokeColor:[UIColor yellowColor]];
+                                    }
+                                }];
+
+self.annotator.dataSendingHandler = ^(NSArray *data, NSError *error) {
+    NSLog(@"%@", data);
+};
+```
+
+### Class design
```

Comment:
Should we add the Class design for TextChat and Core as well?

Reply:
I would say that is out of scope for the merging, but we can create a ticket in general to improve the README. I just copied the other READMEs into here